src/common/settings_objects.cpp

```diff
@@ -120,11 +120,11 @@ namespace PowerToysSettings {
     m_json.GetNamedObject(L"properties").SetNamedValue(name, picker);
   }
 
-  void Settings::add_hotkey(std::wstring_view name, UINT description_resource_id, const HotkeyObject& hotkey) {
+  void Settings::add_hotkey(const std::wstring_view &name,const UINT &description_resource_id, const HotkeyObject& hotkey) {
```

Comment:
nit:  For consistency, do we use `string& var` instead of `string &var`, as the rest of the references are being passed that way. I leave it to your discretion :)

Reply:
the rest of the references? do you mean references in this file or the rest of the code? because in this file all references use (type &variabbleName)

Reply:
I'm probably just being pedantic. I meant with the rest of the PowerToys code.

Reply:
can you please specify where the suggested semantic used used?

Reply:
`const HotkeyObject& hotkey` is one such example. It was not changed by you and is the last parameter of this function add_hotkey(). It is a minor detail and I leave it to you to either change it or leave it as is. The rest of the code LGMT :)